internal/ui/entry_category.go

```diff
@@ -53,6 +53,19 @@ func (h *handler) showCategoryEntryPage(w http.ResponseWriter, r *http.Request)
 
 	entryPaginationBuilder := storage.NewEntryPaginationBuilder(h.store, user.ID, entry.ID, user.EntryOrder, user.EntryDirection)
 	entryPaginationBuilder.WithCategoryID(categoryID)
+
+	// Limit pagination to unread unless 'all' query param is specified
+	if !request.HasQueryParam(r, "all") {
```

Comment:
Actually I think 
```
if request.HasQueryParam(r, "unread") 
```
is easier to read.

Reduce the usage of "Not" "!" .

Reply:
And I don't think there is an agreement on
whether 
`/category/<category_id>/entry/<entry_id>?unread=1`
or 
`/category/<category_id>/unread/entry/<entry_id>`

Reply:
Also `not all` could mean `only unread`. It could also mean `only read` (in case anyone wants it).

So be more specific `only unread` sounds better to me.